test/ingestor.tests.js

```diff
@@ -0,0 +1,61 @@
+const expect = require('chai').expect;
+const { filewalker, state } = require('../ingestor.js');
+
+describe('File Walker Ingestor', () => {
+  const testDirectory = './unzippedArchive';
+  let testState;
+
+  before(done => {
```

Comment:
```suggestion
  beforeEach(() => {
```

You don't need `done`, it is used in promises.

Reply:
This is actually needed for asynchronous code.

Testing asynchronous code with Mocha could not be simpler! Simply invoke the callback when your test is complete. By adding a callback (usually named done) to it(), Mocha will know that it should wait for this function to be called to complete the test. This callback accepts both an Error instance (or subclass thereof) or a falsy value; anything else will cause a failed test.